### PR TITLE
update of cbt recharge

### LINKED ISSUE #65: Recharge at the end of turn
Simple concept: Add a (toggleable) function that allows creatures to roll for their recharge at the _end_ of their turn rather than the start.

This allows you to more dramatically foreshadow the effect, say a dragon's breath.

### PR DESCRIPTION
**Pull request by kandashi**
Update recharge featue to allow for GM rolls for the rechage rolls, and also allow for rolls at the end of a creatures turn rather than the start.
closes #65 
clsoes #68 

@drdwing @vtt-lair @MS-PBS @brnmuller  we have 2 new localization lines (around line 90)
"DND5EH.CombatAbilityRechargeHide_name" : 
"DND5EH.CombatAbilityRechargeHide_hint":"

### CODE REVIEW

File: scripts/dnd5e-helpers.js [hunk 1]
```diff
@@ -150,6 +150,19 @@ Hooks.on('init', () => {
     hint: game.i18n.format("DND5EH.CombatAbilityRecharge_hint"),
     scope: "world",
     config: true,
+    default: "off",
+    type: String,
+    choices: {
+      "off": "Off",
```
Comment (trioderegion) [inline scripts/dnd5e-helpers.js:156]: Needs localization keys

File: lang/es.json [hunk 2]
```diff
@@ -89,6 +89,8 @@
 
     "DND5EH.CombatAbilityRecharge_name":"Hacer las tiradas d6 de recarga para habilidades descargadas automáticamente.",
     "DND5EH.CombatAbilityRecharge_hint":"Activa o desactiva esta función (global)",
+    "DND5EH.CombatAbilityRechargeHide_name" : "Hide the results of ability recharge rolls",
```
Comment (trioderegion) [inline lang/es.json:92]:
```suggestion
    "DND5EH.CombatAbilityRechargeHide_name" : "[EN] Hide the results of ability recharge rolls",
```

File: lang/es.json [hunk 3]
```diff
@@ -89,6 +89,8 @@
...
+    "DND5EH.CombatAbilityRechargeHide_hint":"Enables or disables this feature (global). Can be toggled between start or end of turn",
```
Comment (trioderegion) [inline lang/es.json:93]:
```suggestion
    "DND5EH.CombatAbilityRechargeHide_hint":"[EN] Enables or disables this feature (global). Can be toggled between start or end of turn",
```

File: lang/fr.json [hunk 4]
```diff
@@ -88,6 +88,8 @@
 
     "DND5EH.CombatAbilityRecharge_name":"Lancez automatiquement toutes les capacités non chargées avec une recharge de d6.",
     "DND5EH.CombatAbilityRecharge_hint":"Active ou désactive cette fonctionnalité (global)",
+    "DND5EH.CombatAbilityRechargeHide_name" : "Hide the results of ability recharge rolls",
```
Comment (trioderegion) [inline lang/fr.json:91]:
```suggestion
    "DND5EH.CombatAbilityRechargeHide_name" : "[EN] Hide the results of ability recharge rolls",
```

File: lang/fr.json [hunk 5]
```diff
@@ -88,6 +88,8 @@
...
+    "DND5EH.CombatAbilityRechargeHide_hint":"Enables or disables this feature (global). Can be toggled between start or end of turn",
```
Comment (trioderegion) [inline lang/fr.json:92]:
```suggestion
    "DND5EH.CombatAbilityRechargeHide_hint":"[EN] Enables or disables this feature (global). Can be toggled between start or end of turn",
```

File: lang/ko.json [hunk 6]
```diff
@@ -88,6 +88,8 @@
 
     "DND5EH.CombatAbilityRecharge_name":"모든 D6 재충전 특성 자동 굴림",
     "DND5EH.CombatAbilityRecharge_hint":"이 기능을 활성화/비활성화 한다. (전반적)",
+    "DND5EH.CombatAbilityRechargeHide_name" : "Hide the results of ability recharge rolls",
```
Comment (trioderegion) [inline lang/ko.json:91]:
```suggestion
    "DND5EH.CombatAbilityRechargeHide_name" : "[EN] Hide the results of ability recharge rolls",
```

File: lang/ko.json [hunk 7]
```diff
@@ -88,6 +88,8 @@
...
+    "DND5EH.CombatAbilityRechargeHide_hint":"Enables or disables this feature (global). Can be toggled between start or end of turn",
```
Comment (trioderegion) [inline lang/ko.json:92]:
```suggestion
    "DND5EH.CombatAbilityRechargeHide_hint":"[EN] Enables or disables this feature (global). Can be toggled between start or end of turn",
```

File: lang/pt-BR.json [hunk 8]
```diff
@@ -88,6 +88,8 @@
 
     "DND5EH.CombatAbilityRecharge_name":"Re-rolar automaticamente quaisquer habilidades com uma recarga de d6.",
     "DND5EH.CombatAbilityRecharge_hint":"Habilitar ou desabilitar essa função (global).",
+    "DND5EH.CombatAbilityRechargeHide_name" : "Hide the results of ability recharge rolls",
```
Comment (trioderegion) [inline lang/pt-BR.json:91]:
```suggestion
    "DND5EH.CombatAbilityRechargeHide_name" : "[EN] Hide the results of ability recharge rolls",
```

File: lang/pt-BR.json [hunk 9]
```diff
@@ -88,6 +88,8 @@
...
+    "DND5EH.CombatAbilityRechargeHide_hint":"Enables or disables this feature (global). Can be toggled between start or end of turn",
```
Comment (trioderegion) [inline lang/pt-BR.json:92]:
```suggestion
    "DND5EH.CombatAbilityRechargeHide_hint":"[EN] Enables or disables this feature (global). Can be toggled between start or end of turn",
```

**Review by trioderegion — COMMENTED** [review]
[EN] flags and new localization keys for hiding recharge rolls

File: scripts/dnd5e-helpers.js (hunk 1 above)
Comment (kandashi) [reply scripts/dnd5e-helpers.js]: Added localization for these

**Review by trioderegion — APPROVED** [review]

**Pull request merged**